src/content/overview/architecture/_index.md

```diff
@@ -11,7 +11,77 @@ user_questions:
   - What is the architecture of the Giant Swarm cloud-native developer platform?
 owner:
   - https://github.com/orgs/giantswarm/teams/team-planeteers
-last_review_date: 2024-04-26
+last_review_date: 2024-11-04
 ---
 
-## Platform API {#platform-api}
+Giant Swarm's cloud-native platform is a collection of open-source components that work together to provide a seamless experience for managing the lifecycle of containerized applications. The platform is designed to be cloud-agnostic, allowing you to deploy your applications on any of the supported cloud providers. Besides, the platform offers a rich set of APIs that our customers can use to offer a set of self-service capabilities to their developers.
+
+## Platform architecture
+
+![Platform architecture](./platform-architecture.png)
+
+The platform architecture consists of the following layers:
+
+- Interface layer: offers a set of templates, APIs, and tools that developers can use to interact with the platform.
+
+- Capabilities layer: provides a set of capabilities to manage the lifecycle of applications, including deploying, scaling, and monitoring applications.
+
+- Infrastructure layer: responsible for managing the underlying infrastructure, including the cloud provider resources or on-premises infrastructure. To abstract the underlying infrastructure, the platform uses the `Kubernetes`.
+
+Along these layers, Giant Swarm provides a real support channel, incident management, and good documentation to help you get the most out of the platform. Additionally, you have a dedicated account engineer to help creating a common roadmap and to provide guidance on how to use the platform.
```

Comment:
```suggestion
Along these layers, Giant Swarm provides a live support channel, incident management, and good documentation to help you get the most out of the platform. Additionally, you have a dedicated account engineer to help create a common roadmap and to provide guidance on how to use the platform for your goals.
```

real sounds weird, what would the opposite be "fake"?

Reply:
+1 for "live", or even 24/7 since together with incident handling, the "channel" is basically all day and night